Vostok.ZooKeeper.Recipes/Helpers/DistributedLockToken.cs

```diff
@@ -70,5 +70,29 @@ public void Dispose()
                 delete.EnsureSuccess();
             }
         }
+        
+#if NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_0_OR_GREATER
+        public async ValueTask DisposeAsync()
+        {
+            if (disposed.TrySetTrue())

```

Comment:
Code fully duplicated from Dispose(). It's better to extract method and reuse it both in Dispose and DisposeAsync